programs/nft-voter/src/state/nft_vote_record.rs

```diff
@@ -0,0 +1,84 @@
+use anchor_lang::prelude::*;
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+use solana_program::program_pack::IsInitialized;
+
+use spl_governance_tools::account::{get_account_data, AccountMaxSize};
+
+use crate::{error::NftVoterError, id};
+
+/// Vote record indicating the given NFT voted on the Proposal
+/// The PDA of the record is ["nft-vote-record",proposal,nft_mint]
+/// It guarantees uniques and ensures the same NFT can't vote twice
+#[derive(Clone, Debug, PartialEq, BorshDeserialize, BorshSerialize, BorshSchema)]
```

Comment:
nit: Can probably use bytemuck here to avoid copies, it's all arrays essentially

Reply:
Do you mean the zero copy option in Anchor or something else?

Reply:
Yes, I think Anchor's "zero-copy" mode just uses bytemuck underneath